### DISCUSSION TITLE
Badge Label is All Caps

### DISCUSSION THREAD

**Discussion by kimfucious** [mantinedev/mantine#2411] [Feature requests]
At present, I can't see a way to change a Badge label to not be all caps.

If there's a way to do this, please let me know; otherwise, I think this would be a feature that people might find of value.

Thanks!

**Comment by rtivital** [comment]
https://codesandbox.io/s/adoring-sara-m9bwrl?file=/src/App.tsx

**Reply by kimfucious** [reply]
Perfect, awesome, thank you.

I should have thought of that myself, sorry.

Might be nice to make this a prop.
